src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -168,6 +172,8 @@ public void EnsureTaskNameIsNotUsed(string taskName)
 
             if (ExternalReplications.Any(x => x.Name.Equals(taskName, StringComparison.OrdinalIgnoreCase)))
                 throw new InvalidOperationException($"Can't use task name '{taskName}', there is already an External Replications task with that name");
+            if (SinkPullReplications.Any(x => x.Name.Equals(taskName, StringComparison.OrdinalIgnoreCase)))
```

Comment:
`HubPullReplications` is dictionary so we don't need to worry about duplicate names.

Reply:
What you are doing is making sure that there aren't duplicated names across different types of stuff.
sink pull, external repl, etc.
It isn't for correctness, it is to avoid confusion with the user